Source/Scene/ModelInstanceCollection.js

```diff
@@ -54,6 +56,34 @@ define([
         FAILED : 3
     };
 
+    function ModelInstance(collection, modelMatrix, instanceId) {
```

Comment:
Should this be `@private` and in another file since it can be used outside of this file?